aksel.nav.no/website/pages/eksempler/actionmenu/sub-menu.tsx

```diff
@@ -0,0 +1,139 @@
+import { MenuElipsisVerticalIcon } from "@navikt/aksel-icons";
+import { ActionMenu, Button, Table } from "@navikt/ds-react";
+import { withDsExample } from "@/web/examples/withDsExample";
+
+const Example = () => {
+  return (
+    <Table>
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell scope="col">ID</Table.HeaderCell>
+          <Table.HeaderCell scope="col">Status</Table.HeaderCell>
+          <Table.HeaderCell aria-label="" />
```

Comment:
What does this empty `aria-label` do?

Reply:
Was something Morten wanted to try since it's technically an a11y violation. But as seen in the screenshot, it looks like empty aria-labels were not valid so just removed it. Morten said that its not an actual issue with empty header
<img width="498" alt="Screenshot 2024-10-09 at 12 09 17" src="https://github.com/user-attachments/assets/4a373942-c716-482e-97ae-592ff86b6eeb">

Reply:
It came back :p